src/Symfony/Component/ObjectMapper/TransformCallableInterface.php

```diff
@@ -25,6 +26,7 @@ interface TransformCallableInterface
     /**
      * @param mixed $value  The value being mapped
      * @param T     $source The object we're working on
+     * @param T2|null $target The target we're mapping to
```

Comment:
I don't think we can specify the generic type here. The function can receive any object.

Reply:
You can when you now the source and the target types

Reply:
Ok, maybe.